addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -438,7 +438,7 @@ def _process_step_forward_operator(self, discuss_channel, users=None):
 
     def _to_store_defaults(self):
         return [
-            Store.Many("answer_ids", rename="answers"),
+            Store.Many("answer_ids"),
```

Comment:
`Many` is not required anymore.

Reply:
How so? Many is not required when providing just ids "Many(..., [])" but the current syntax will call the to store of the related model, not adding the Many will just add the ids.

Reply:
We figured that after trying, we added `Many` back.
I don't see the added value to return only `ids`, it would be easier if the store did its best to always provide the values.

Reply:
The confusion comes from the magic of store default, which I want to remove. It would be best if we always simply listed the fields we need (they can come from a common method returning them if the same list is used in several places).